invenio_app_ils/ui/src/semantic-ui/site/collections/menu.overrides

```diff
@@ -6,10 +6,23 @@
     border-top: @highlightBorder;
 }
 
-.ui.menu.mobile-header-menu{
+.ui.menu.mobile-header-menu {
   display: block;
 }
 
+.fs-search-controls-mobile {
```

Comment:
nit: I would wrap `fs` and `bo` classes inside the main `frontsite`/`backoffice`.
```
.frontsite {
     .fs-search-controls-mobile { }
}
```